pages/_app.tsx

```diff
@@ -63,6 +64,7 @@ const WrappedApp: FC<AppProps> = ({ Component, pageProps }) => {
 
   return (
     <Layout>
+      <GTagScript />
```

Comment:
I don't believe this should be here.
It needs to be in the <Head> in the `_document.tsx` file